### DISCUSSION TITLE
Late binding with lambda

### DISCUSSION THREAD

**Discussion by CatamountJack** [zauberzeug/nicegui#1171] [Ideas / Feature Requests]
Seems like the late binding issue is a common problem for folks using lambda to pass arguments to events.  Would it be worth documenting that somewhere on the Documentation page?  I'm not sure where it best fits since it applies to many types of elements; maybe a section of its own?  I don't have time to come up with a PR at the moment, but just throwing the idea out there for now.

**Comment by falkoschindler** [comment]
That's a very good point, @CatamountJack!

I thought about it now and than, but struggle with the right place to put it as well.
And: Someone new to the concept wouldn't even know how to search for it. Even I myself learned the term "late binding" only weeks ago.

But I'm very much looking forward to suggestions how to approach this. 🙂

**Comment by rodja** [comment]
Maybe we could introduce a FAQ section where things like this more general Python advices have a place.

**Reply by falkoschindler** [reply]
Good idea. Even if the user doesn't find it, we can point to it.

**Comment by rodja** [comment]
I just created a first FAQ with a short explanation of late binding: https://github.com/zauberzeug/nicegui/wiki/FAQs#why-have-all-my-elements-the-same-value
